emscripten.py

```diff
@@ -1800,8 +1800,10 @@ def create_fp_accessors(metadata):
     # the name of the original function is generally the normal function
     # name, unless it is legalized, in which case the export is the legalized
     # version, and the original provided by orig$X
+    originalAccessor = ''
     if shared.Settings.LEGALIZE_JS_FFI and not shared.JS.is_legal_sig(sig):
       name = 'orig$' + name
+      originalAccessor = ('if (!func) func = Module["_%s"];' % (name))
```

Comment:
Remove the extra pair of (  ) around this statement.

Reply:
Sure